client/views/network.tpl

```diff
@@ -1,5 +1,11 @@
 {{#each networks}}
-<section id="network-{{id}}" class="network name-{{slugify name}} {{#if serverOptions.NETWORK}}network-{{slugify serverOptions.NETWORK}}{{/if}}" data-id="{{id}}" data-nick="{{nick}}" data-options="{{tojson serverOptions}}">
+<section
+	id="network-{{id}}"
+	class="network name-{{slugify name}} {{#if serverOptions.NETWORK}}network-{{slugify serverOptions.NETWORK}}{{/if}} {{#unless status.connected}}not-connected{{/unless}} {{#unless status.secure}}not-secure{{/unless}}"
+	data-id="{{id}}"
+	data-nick="{{nick}}"
+	data-options="{{tojson serverOptions}}"
+>
```

Comment:
I think we should have a tooltip (opening towards the right, "east" side) for not-secure and not-connected types, to inform the user about what those icons / colors mean.

Reply:
It would probably be a pain to get tooltips in there correctly since the icons are on `:before` of the entire title.

Reply:
Yeah, I tried and quickly gave up...